### PR TITLE
CalculateInsideWhichField module

### LINKED ISSUE #2019: CalculateInsideWhichField module
CalculateInsideWhichField module needs the ability to set the Default Outside Value to NaN

### PR DESCRIPTION
**Pull request by nids2001**
closes #2019

### CODE REVIEW

**Comment by dcwhite** [comment]
@nids2001 Cool. Please add a new test network to `ExampleNets/regression/modules` that we can use to test this out. If there's a `ReportFieldInfo` connected to the output, that's probably enough.

**Comment by dcwhite** [comment]
Thanks!

File: src/Interface/Modules/Fields/CalculateInsideWhichFieldDialog.ui [hunk 1]
```diff
@@ -254,6 +254,19 @@
...
+          <horstretch>0</horstretch>
+          <verstretch>0</verstretch>
+         </sizepolicy>
+        </property>
+        <property name="text">
+         <string>Use NaN</string>
+        </property>
+       </widget>
+      </item>
+      <item row="5" column="2">
```
Comment (dcwhite) [inline src/Interface/Modules/Fields/CalculateInsideWhichFieldDialog.ui]: One more thing to add to the UI. Check out this part of the related UI, when the Nan button is clicked, the spinner is disabled:
https://github.com/SCIInstitute/SCIRun/blob/master/src/Interface/Modules/Fields/MapFieldDataOntoElems.ui#L248

Comment (dcwhite) [reply src/Interface/Modules/Fields/CalculateInsideWhichFieldDialog.ui]: This can be added inside Qt Designer using the Signal/Slot interface.

Comment (allywarner) [reply src/Interface/Modules/Fields/CalculateInsideWhichFieldDialog.ui]: When you click the NaN checkbox, the default outside value is set to 999999999.00. Is this expected? The default value is 0.00 when opening the UI. When you uncheck the NaN checkbox, the number stays 999999999.00. Should it go back to the number it was before checking the box?

Comment (dcwhite) [reply src/Interface/Modules/Fields/CalculateInsideWhichFieldDialog.ui]: I think that's the behavior of the other modules with similar NaN options. We can think of a smoother transition later.

**Pull request merged**